### PR TITLE
Fix error while generating pdf when venue is nil

### PR DESCRIPTION
**Pull request by rahul2240**
Error in generating pdf for ticket when venue is not set is fixed.Showing venue in pdf only if venue exist.
Closes https://github.com/opensuse/osem/issues/1683

### CODE REVIEW

**Comment by rahul2240** [comment]
Can you tell me why only venue's name is mentioned in pdf?

File: app/pdfs/ticket_pdf.rb [hunk 1]
```diff
@@ -51,7 +51,9 @@ def draw_second_square
     move_down 70
     draw_text @conference.title.to_s, at: [@mid_horizontal + 30, cursor - 30], size: 12
     draw_text @conference.organization.name.to_s, at: [@mid_horizontal + 30, cursor - 50], size: 12
-    draw_text @conference.venue.name.to_s, at: [@mid_horizontal + 30, cursor - 70]
+    if !@conference.venue.nil?
```
Comment (differentreality) [inline app/pdfs/ticket_pdf.rb]: Why not just `if conference.venue` ?

**Comment by differentreality** [comment]
Hmmm perhaps @siddhantbajaj or @Ana06 can shed some light on that question. 
I assume you mean why it doesn't also show its address, right? I would agree that we should be also showing the full address of the venue on the ticket.

**Comment by siddhantbajaj** [comment]
@rahul2240 Thanks for the PR. Yes I think we should indeed show the full address in the ticket. Will you take this up as well in this PR? If yes please add the screenshot of your changes. 😉

File: app/pdfs/ticket_pdf.rb [hunk 2]
```diff
@@ -51,7 +51,9 @@ def draw_second_square
...
+      draw_text @conference.venue.name.to_s, at: [@mid_horizontal + 30, cursor - 70]
```
Comment (Ana06) [inline app/pdfs/ticket_pdf.rb]: `to_s` shouldn't be needed, right? `name` is already an string (I know this was already there, so you can add the change in a different commit) :wink:

**Comment by rahul2240** [comment]
@siddhantbajaj Please review the changes.
ticket

File: app/pdfs/ticket_pdf.rb (hunk 2 above)
Comment (rahul2240) [reply app/pdfs/ticket_pdf.rb]: @Ana06 Sure! :smile:

**Comment by Ana06** [comment]
@rahul2240 please rebase, the Travis run is broken, but in master is already fixed :wink:

**Comment by rahul2240** [comment]
@Ana06 Sorry i forgot to rebase, travis is working now :smile:

**Comment by rahul2240** [comment]
@Ana06 can you close this PR if it is fixed?

**Comment by differentreality** [comment]
Thank you @rahul2240  :+1: 

I don't see anything else pending for this PR, and travis is passing, so merging this :tada:

**Pull request merged**